Incompatability with certain mods in a modpack

I have this same exact problem.
---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 4/16/17 6:54 PM
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.NoSuchMethodError: net.minecraftforge.oredict.OreDictionary.doesOreNameExist(Ljava/lang/String;)Z
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:162)
	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:692)
	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:288)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:541)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoSuchMethodError: net.minecraftforge.oredict.OreDictionary.doesOreNameExist(Ljava/lang/String;)Z
	at mods.eln.Eln.firstExistingOre(Eln.java:7534)
	at mods.eln.Eln.recipeTurbine(Eln.java:5667)
	at mods.eln.Eln.load(Eln.java:768)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:187)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:691)
	... 10 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_71, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 138322888 bytes (131 MB) / 529899520 bytes (505 MB) up to 4281597952 bytes (4083 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.82.1216 Minecraft Forge 10.13.1.1216 Optifine OptiFine_1.7.10_HD_U_C1 20 mods loaded, 20 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized
	FML{7.10.82.1216} [Forge Mod Loader] (forge-1.7.10-10.13.1.1216-new.jar) Unloaded->Constructed->Pre-initialized->Initialized
	Forge{10.13.1.1216} [Minecraft Forge] (forge-1.7.10-10.13.1.1216-new.jar) Unloaded->Constructed->Pre-initialized->Initialized
	ArchitectureCraft{1.4.0} [ArchitectureCraft] (ArchitectureCraft-1.4.0-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized
	CarpentersBlocks{3.3.7} [Carpenter's Blocks] (Carpenter's Blocks v3.3.7 - MC 1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized
	Eln{@VERSION@} [Electrical Age] (ElectricalAge-1.14.2.jar) Unloaded->Constructed->Pre-initialized->Errored
	flenixcities{0.11.1} [FlenixCities] (FlenixCitiesCore_[1.7.10]-0.11.1-b97.jar) Unloaded->Constructed->Pre-initialized->Initialized
	fc_research{0.4.0} [FlenixCities Research] (FlenixCitiesResearch_[1.7.10]-0.4 (1).jar) Unloaded->Constructed->Pre-initialized->Initialized
	GregsLighting{1.11.2} [Greg's Lighting] (GregsLighting-1.11.2-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized
	malisiscore{1.7.10-0.12.5} [MalisisCore] (malisiscore-1.7.10-0.12.5.jar) Unloaded->Constructed->Pre-initialized->Initialized
	malisisdoors{1.7.10-1.9.3} [Malisis' Doors] (malisisdoors-1.7.10-1.9.3.jar) Unloaded->Constructed->Pre-initialized->Initialized
	Minema{v3.0-1.7.10} [Minema] (Minema-Mod-1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized
	cfm{3.4.7} [§9MrCrayfish's Furniture Mod] (MrCrayfishFurnitureModv3.4.7(1.7.10).jar) Unloaded->Constructed->Pre-initialized->Initialized
	musiccraft{2.9.8} [MusicCraft] (musiccraft-2.9.8.jar) Unloaded->Constructed->Pre-initialized->Initialized
	NGTLib{1.7.10.4} [NGTLib] (NGTLib-1.7.10.zip) Unloaded->Constructed->Pre-initialized->Initialized
	RadixCore{2.1.3} [RadixCore] (RadixCore-1.7.10-2.1.3-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized
	RTM{1.7.10.5} [Real Train Mod] (Real-Train-Mod-1.7.10 (2).zip) Unloaded->Constructed->Pre-initialized->Initialized
	secretroomsmod{4.7.1} [The SecretRoomsMod] (secretroomsmod-1.7.10-4.7.1.394.jar) Unloaded->Constructed->Pre-initialized->Initialized
	SlimevoidLib{2.0.4.7} [Slimevoid Library] (SlimevoidLibrary-2.0.4.7.jar) Unloaded->Constructed->Pre-initialized->Initialized
	worldedit{6.0-beta-01} [WorldEdit] (worldedit-forge-mc1.7.10-6.0-beta-01.jar) Unloaded->Constructed->Pre-initialized->Initialized

--- a/build.gradle
+++ b/build.gradle
@@ -56,6 +56,8 @@ minecraft {
     // TODO(Baughn): After moving to 1.10, this should be replaced with the shadow plugin,
     // e.g. as used in https://github.com/Emberwalker/Laundarray/blob/master/build.gradle
     srgExtra "PK: org/apache/commons/math3 mods/eln/libs/org/apache/commons/math3"
+    srgExtra "PK: kotlin mods/eln/libs/kotlin"
+    srgExtra "PK: org/jetbrains/annotations mods/eln/libs/annotations"
 
     replaceIn "Version.java"
     replace "@VERSION@", project.version
@@ -67,6 +69,8 @@ minecraft {
 configurations {
     external
     compile.extendsFrom external
+    shade
+    compile.extendsFrom shade
 }
 
 repositories {
@@ -83,8 +87,8 @@ repositories {
 
 dependencies {
     external files("libs/commons-math3-3.3.jar")
-    external "org.jetbrains.kotlin:kotlin-runtime:$kotlin_version"
-    external "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    shade "org.jetbrains.kotlin:kotlin-runtime:$kotlin_version"
+    shade "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compile "mcp.mobius.waila:Waila:1.5.11-RC2-NONEI_1.7.10:dev"
 }
 
@@ -153,6 +157,12 @@ jar {
     from('src/main/resources/assets/eln/textures') {
         include '**/*.png'
         into 'assets/eln/textures'
+    }
+    
+    configurations.shade.each { dep ->
+        from(project.zipTree(dep)){
+            exclude 'META-INF', 'META-INF/**'
+        }
     }
 }
 

